presto-spi/src/main/java/com/facebook/presto/spi/tracing/TracerProvider.java

```diff
@@ -19,5 +19,5 @@
      *
      * @return A @Tracer that should be kept throughout the whole duration of tracing.
      */
-    Tracer getNewTracer();
+    Tracer getNewTracer(String tracerName, String traceToken);
```

Comment:
is there a reason we need to pass in a `tracerName` if the instance of the tracer already determines that information?

Reply:
also, are we getting a new tracer each time; is this why we are passing in the trace token?

Reply:
we have a singular tracer for all http requests. however, a new span is created for each request / query and they have a trace token attribute which is why we need to pass in `traceToken`

Reply:
we pass in `tracerName` because if we call this method from different modules, we want different tracers created. if we call getNewTracer with the same tracerName, it will return the same Open Telemetry tracer